vm/src/stdlib/posix.rs

```diff
@@ -1228,7 +1228,7 @@ pub mod module {
     }
 
     #[pyfunction]
-    fn sync() {
+    const fn sync() {
```

Comment:
#[pyfunction], #[pymethod], #[pygetset] etc will have no change to be called in const context. you don't need to spend too much effort about them. 

`const fn` to rust-context functions is a good idea

Reply:
> #[pyfunction], #[pymethod], #[pygetset] etc will have no change to be called in const context. you don't need to spend too much effort about them.
> 

It's a simple clippy command:)